### PR TITLE
Bump version to 0.19.13

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

**Comment by quisquous** [comment]
@valarnin Can you merge this after approval? I mostly want to fix the fate issue because it creates some obnoxious behavior (all fates removed are treated as added when you leave the zone and the noises all go off at once).

**Review by valarnin — APPROVED** [review]

**Pull request merged**
